plugins/kotlin-as-java/src/main/kotlin/converters/KotlinCompanion.kt

```diff
@@ -0,0 +1,78 @@
+package org.jetbrains.dokka.kotlinAsJava.converters
+
+import org.jetbrains.dokka.links.Callable
+import org.jetbrains.dokka.model.*
+import org.jetbrains.dokka.model.properties.PropertyContainer
+
+private const val DEFAULT_COMPANION_NAME = "Companion"
+
+internal class KotlinCompanion(private val companion: DObject?) {
+
+    fun staticFunctions(): List<DFunction> {
+        if(companion == null) return emptyList()
+        return companion.functions.filter { it.isJvmStatic }
+    }
+
+    /**
+     * @return properties that will be visible as static for java.
+     * See [Static fields](https://kotlinlang.org/docs/java-to-kotlin-interop.html#static-fields)
+     */
+    fun staticProperties(): List<DProperty> {
```

Comment:
I'd rename it to something like `staticPropertiesForJvm` as there is no entity as static in Kotlin and can be slightly confusing